debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/DatabaseDifferences.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql;
+
+/**
+ * A centralized expression of differences in behaviour between MySQL 5.x and 8.x
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public interface DatabaseDifferences {
```

Comment:
Nice idea! I don't think so that methods specific to a single test should be here. I'll hence move the last three which are related only to the single geo-spetial test class and its methods to a separate one.